quickwit-cli/src/index.rs

```diff
@@ -241,9 +249,44 @@ impl IndexCliCommand {
         }))
     }
 
-    fn parse_demux_args(_matches: &ArgMatches) -> anyhow::Result<Self> {
-        // TODO
-        Ok(Self::Demux(DemuxArgs {}))
+    fn parse_merge_args(matches: &ArgMatches) -> anyhow::Result<Self> {
+        let metastore_uri = matches
+            .value_of("metastore-uri")
+            .context("'metastore-uri' is a required arg")
```

Comment:
```suggestion
            .context("'metastore-uri' is a required arg.")
```
I think we have it in other places too